applications/callflow/src/cf_route_win.erl

```diff
@@ -279,19 +279,32 @@ maybe_start_metaflow(Call) ->
 -spec maybe_start_metaflow(kapps_call:call(), kz_term:api_binary()) -> kapps_call:call().
 maybe_start_metaflow(Call, 'undefined') ->
     maybe_start_endpoint_metaflow(Call, kapps_call:authorizing_id(Call)),
+    maybe_start_ts_metaflow(Call, kapps_call:authorizing_id(Call)),
     Call;
 maybe_start_metaflow(Call, App) ->
     lager:debug("metaflow app ~s", [App]),
     Call.
 
+-spec maybe_start_ts_metaflow(kapps_call:call(), kz_term:api_binary()) -> 'ok'.
```

Comment:
Metaflows don't really seem appropriate to apply to a downstream PBX system. Wouldn't that system be the one offering (or not) the ability to take action on DTMFs? Trunkstore really only concerns itself with trunking a simple failover.

Reply:
trunstore + metaflow may used if we want move functionality to kazoo from old PBX.
I used this functionality only for one customer and not sure that will be used for other.
I can simple close this PR as not wanted.